layouts/header.html

```diff
@@ -40,6 +40,7 @@
 
     <!-- Custom styles for this template -->
     <link href="/css/docs.css" rel="stylesheet">
+    <link href="/css/routing-tree-editor.css" rel="stylesheet">
```

Comment:
This is now used in every page. Don't know whether there's a decent way to do it differently with nanoc, so fine with me. 
Anyone bothered by that?